### DISCUSSION TITLE
Transform property changes when scaling a reference element

### DISCUSSION THREAD

**Discussion by chmilvs** [floating-ui/floating-ui#3111] [General]
Hello! I use floating ui in the react application. Please tell me how to avoid jumping the floating element if the reference element uses transform: scale? Because of this, transform:translate from the floatingStyles changes of the floating element
Thank you!

https://github.com/user-attachments/assets/fb9fe2e2-08ae-466e-be77-880a5ce04475

**Comment by atomiks** [comment]
Can disable `layoutShift` in `autoUpdate`: https://floating-ui.com/docs/autoUpdate#layoutshift

**Reply by chmilvs** [reply]
It worked, thank you very much!
